dashboard/app/helpers/levels_helper.rb

```diff
@@ -50,14 +50,14 @@ def readonly_view_options
     view_options callouts: []
   end
 
-  def set_channel
+  def set_channel(level)
```

Comment:
@balderdash  I think I see what you are saying here, but I am concerned that this change makes the LevelsHelper class harder to understand. I think we need to choose between:
1. a helper class which operates on `@level`, except when explicitly overridden (i.e. Elijah's comment)
2. a helper class which is initialized with a level, which then operates on that level and whose methods don't take a level param (not sure how well this works in the rails / controller / helper world)

My reasoning is that it's confusing that the `level` param is required param in almost every method, but that `@level` is still referenced directly in some methods. And if you're going to require the `level` param in every method, then it would be better to supply it when you initialize the class.